CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 * Improved documentation about `dynamicType:` vs `type(of:)`.  
   [Antondomashnev](https://github.com/Antondomashnev)
   [#221](https://github.com/AliSoftware/OHHTTPStubs/pull/221)
+* Fixed a race condition that occasionally prevented redirect callbacks. Fixed response timing for zero-length stub data.  
```

Comment:
If I'm being pedantic, I think it would read more clearly with two separate entries/bullet points instead of one 😝